src/ncurses/window.cr

```diff
@@ -13,8 +13,16 @@ module NCurses
       initialize(LibNCurses.newwin(height || max_height, width || max_width, y, x))
     end
 
+    def get_max_x
+      LibNCurses.getmaxx(self)
+    end
+
+    def get_max_y
+      LibNCurses.getmaxy(self)
+    end
+
     def max_dimensions
-      {LibNCurses.getmaxy(self), LibNCurses.getmaxx(self)}
+      {get_max_y, get_max_x}
```

Comment:
i'd prefer a named tuple. so: `{ x: max_x, y: max_y }` to avoid confusion.

Reply:
cooler would actually be a record `Point` with attributes `x` and `y` for type safety.

Reply:
Good idea, but, I'm not sure to fully understand how you see that:
- Where should I create this Point struct (namespace, visibility).
- Next in the following comments ;).

Reply:
For now, I push the raw changes that you asked. Maybe we can add your Point idea in another PR ?